src/PHPUnit/HttpMockFacade.php

```diff
@@ -84,4 +90,60 @@ public function each(callable $callback)
     {
         $callback($this);
     }
+
+    /**
+     * Get error output (if exists)
+     *
+     * @return string
+     */
+    public function getErrorOutput()
+    {
+        return static::cleanErrorOutput($this->server->getIncrementalErrorOutput());
+    }
+
+    /**
+     * Get error output (if exists)
+     *
+     * @param string $output
+     *
+     * @return string
+     */
+    public static function cleanErrorOutput($output)
```

Comment:
🎨 Maybe this should move to an error filter class?

Reply:
You are right escpecially the static call in AppIntegrationTest is ugly...but I would prefer moving it to the Server too (non static)